packages/react-data-grid/src/RowGroup.tsx

```diff
@@ -27,10 +28,10 @@ interface Props {
   name: string;
   renderer?: React.ComponentType;
   eventBus: EventBus;
-  renderBaseRow(p: RowRendererProps): React.ReactElement;
+  renderBaseRow(p: RowRendererProps<R>): React.ReactElement;
 }
 
-const RowGroup = forwardRef<HTMLDivElement, Props>(function RowGroup(props, ref) {
+const RowGroup = forwardRef<HTMLDivElement, Props<any>>(function RowGroup(props, ref) {
```

Comment:
I do not think there is a way to specify generic parameters for forwardRef components. I will revisit this